scripts/push_test_image.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+# Copyright 2022 Google LLC
```

Comment:
nit: 2024 and elsewhere

Reply:
Done in https://github.com/datacommonsorg/website/pull/4175